docs/prompting/zero_shot/emotion_prompting.md

```diff
@@ -1,7 +1,46 @@
 ---
-title: ""
+title: "Emotion Prompting"
 description: ""
-keywords: ""
+keywords: "prompt, prompting, prompt engineering, llm, gpt, openai, emotion prompting, model, AI, python, instructor, zero-shot"
 ---
 
-[wip]
+Emotion prompting<sup><a href="https://arxiv.org/abs/2307.11760">1</a></sup> uses phrases that have psychological relevance to humans in the LLM prompt. This may lead to improved performance on benchmarks and open-ended text generation.<sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>
+
+```python
+import openai
+import instructor
+from pydantic import BaseModel
+
+class Recipe(BaseModel):
+    dish_name: str
+    ingredients: list[str]
+    steps: list[str]
+
+client = instructor.from_openai(openai.OpenAI())
+
+emotion_prompt = {
+    "role": "user",
+    "content": (
+        "I want to impress my friends with my cooking." # Emotion prompting
+        "Provide a recipe for a delicious chocolate cake."
+    )
+}
+
+recipe = client.chat.completions.create(
+    model="gpt-4o",
+    response_model=Recipe,
+    messages=[emotion_prompt],
```

Comment:
Don't use variables for the messages just have them directly in because we don't reuse these variables. It just makes reading it very jumpy.